Upgrade to v7 and refactor docs (#60)
PureStake/moonbeam-docs: getting-started/local-node/using-metamask.md

## Connect to the Local Moonbeam Node

> Now, let’s connect MetaMask to our locally running Moonbeam node, which will begin to produce blocks as they are authored:

Reviewer: ```suggestion
Now, let’s connect MetaMask to our locally running Moonbeam node, which will begin to author blocks as transactions arrive:
```

Author: Thank you!!
